snippets/Google.Datastore.V1Beta3.Snippets/DatastoreDbSnippets.cs

```diff
@@ -166,15 +166,15 @@ public void GqlQuery()
             string projectId = _fixture.ProjectId;
             string namespaceId = _fixture.NamespaceId;
 
-            // Snippet: RunQueryAsync(GqlQuery,*,*)
+            // Snippet: RunGqlQueryAsync(GqlQuery,*,*)
```

Comment:
Humbug, yes. I've gone back and forth on it, and must have adjusted it *after* running the snippet extractor for the last time (which would have caught this).